src/query/interpreter.hpp

```diff
@@ -515,7 +515,7 @@ std::map<std::string, TypedValue> Interpreter::Pull(TStream *result_stream, std:
     query_execution.reset(nullptr);
     throw;
   } catch (const utils::BasicException &) {
-    EventCounter::IncrementCounter(EventCounter::FailedQuery);
+    memgraph::metrics::IncrementCounter(memgraph::metrics::FailedQuery);
```

Comment:
If the semantic of failed query is wrongly typed, then it seems to me this is wrong but please explain it to me :)

Reply:
I'm not entirely sure at the moment as I just adopted this as part of the counters that were already present in the system, but let me check quickly.

Reply:
Did you check maybe?

Reply:
Yes I see, I was wrong about the semantics. In the EventCounter it says: "Number of times executing a query failed"

That's why it is in: `interpreter.cpp::Prepare` (because it can be a semantic or a syntax exception or any exception while preparing the query), and in `interpreter.hpp::Pull` for query runtime exceptions

Reply:
:+1: